SPEC.md

```diff
@@ -306,8 +306,7 @@ The test index makes use of symbolic language in describing connection and messa
 
 ### ZG-CONFORMANCE-026
 
-    The test client should send a valid mtVALIDATORLIST message, with both master and signature public keys, correctly serializing a manifest and validator blob.
-    The create is done, but the result is not yet checked.  That will be done in a future task/PR.
+    The test client should send a valid mtVALIDATORLIST message, with both master and signature public keys, correctly serializing a manifest and validator blob.  A second node verifies the correctness by receiving the validator list in an mtVALIDATORLISTCOLLECTION message and verifying the contents.
```

Comment:
I checked other SPEC test descriptions, and there is no need to mention `the test` here,
maybe something like this (so it is more aligned with other test descriptions):
```suggestion
    A synthetic node sends a mtVALIDATORLIST message with both master and signature public keys, correctly serializing a manifest and validator blob to the node.  To verify the node has received the message, another synthetic node awaits a mtVALIDATORLISTCOLLECTION message from the node with the same validator blob sent by the first synthetic node in its mtVALIDATORLIST message. 
```

Reply:
done